knora-ontologies/standoff-onto.ttl

```diff
@@ -97,7 +99,19 @@
                               owl:maxCardinality "1"^^xsd:nonNegativeInteger
```

Comment:
Shouldn't a hyperlink have exactly one reference attribute?
I guess I did this wrong.